WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php

```diff
@@ -104,6 +115,16 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
+		if ( $this->tokens[ $nextToken ]['code'] === T_CONSTANT_ENCAPSED_STRING && filter_var( str_replace( [ '"', "'" ], '', $this->tokens[ $nextToken ]['content'] ), FILTER_VALIDATE_URL ) ) {
```

Comment:
Why did you move this check up ?

Reply:
I moved it up so that it doesn't bail before checking for an external URL with a keyword from `$customPaths` in it. e.g.:

```
include( "http://path.com/bad_file.php" );
```